src/client/components/Story/StoryFull.js

```diff
@@ -462,7 +463,18 @@ class StoryFull extends React.Component {
           />
         )}
         <div className="StoryFull__topics">
-          {tags && tags.map(tag => <Topic key={tag} name={tag} />)}
+          <Scrollbars
+            universal
+            autoHide
+            renderView={({ style, ...props }) => (
+              <div style={{ ...style, marginBottom: '-20px' }} {...props} />
```

Comment:
`react-custom-scrollbars` uses native track and thumb on mobile, so it is not customizable. This hides them.

Reply:
correct me if im wrong but isn't style the only thing being passed here for `renderView` ?

https://github.com/malte-wessel/react-custom-scrollbars/blob/master/src/Scrollbars/index.js#L574

So dont need the `{...props}` right?

Reply:
They use props here as well:
https://github.com/malte-wessel/react-custom-scrollbars/blob/master/examples/simple/components/ColoredScrollbars/ColoredScrollbars.js#L19

Reply:
ahh i see ok